test/csv/test_csv_filter.rb

```diff
@@ -137,4 +137,19 @@ def test_option_col_sep
     assert_equal(["aaa.bbb.ccc\nddd.eee.fff\n", ""],
                  run_csv_filter(csv, "--col-sep=:", "--output-col-sep=."))
   end
+
+  def test_option_quote_char
+    quote_char = "'"
+    csv = CSV.generate(quote_char: quote_char) do |csv|
+      csv << ['foo', 0]
+      csv << ["'bar'", 1]
+      csv << ['"baz"', 2]
+    end
+    assert_equal(["foo,0\n'''bar''',1\n\"baz\",2\n", ""],
+                 run_csv_filter(csv, "--quote-char=:"))
```

Comment:
Is `--quote-char=:` a good test?
It does nothing, right?

Reply:
I think we have the same situation for `--col-sep=:` and `--row-sep=:`, don't we?  They all at least test that the option is accepted?

Reply:
Really?

They use `:` in the input CSV:

https://github.com/ruby/csv/blob/7eea19a639d6c03829b337e5c916804d8e1c10cc/test/csv/test_csv_filter.rb#L132-L134

https://github.com/ruby/csv/blob/7eea19a639d6c03829b337e5c916804d8e1c10cc/test/csv/test_csv_filter.rb#L122-L124